public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -205,7 +205,18 @@ export class ElasticDatasource
       indexList = [this.indexPattern.getIndexForToday()];
     }
 
-    const indexUrlList = indexList.map((index) => index + url);
+    // make sure `url` does not start with a slash
+    url = url.replace(/^\//, '');
```

Comment:
I mean, we can, but shouldn't we then also hardcode `requestAllIndices` to not accept an `url` parameter? Because what if the next person/developer wants to use `requestAllIndices` at a different place.

I removed it in 3367a8ce3e8ec06a5e527deda3f64696cdd428ef but would still leave the detection and "normalization" as part of `requestAllIndices`.

Reply:
> but shouldn't we then also hardcode requestAllIndices to not accept an url parameter?


I think that it make sense then. And it is more clear and understandable.

Reply:
Updated in d46481102a2d7d8b6c395a3d85b27fa7b022f98d